Modules/core/Model/CoreSpaceUser.php

```diff
@@ -54,6 +54,20 @@ public function exists($id_user, $id_space){
         return false;
     }
     
+    /**
+     * 
+     * Returns true if user is an active member of any space
+     * 
+     * @param int $id_user
+     * 
+     * @return bool
+     */
+    public function isActiveMemberOfAnySpace($id_user){
```

Comment:
There is already a method to get user role, if result is null, then there is no role.
Should be enough rather than creating a new method for that